src/screen/profile/index.tsx

```diff
@@ -1,129 +1,149 @@
-                <View style={styles.body}>
-                  <Icon name="heart-sharp" size={29} color={Colors.mediumGrey} />
-                  <Text style={styles.sectionText}>Wishlist</Text>
-                </View>
-                <Icon
-                  name="ios-chevron-forward"
-                  size={25}
-                  style={styles.sectionIcon}
-                />
-              </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.sectionBody}
+          onPress={() => {
+            navigation.navigate('wishlist');
+          }}>
+          <View style={styles.body}>
+            <Icon name="heart-sharp" size={29} color={Colors.mediumGrey} />
+            <Text style={[styles.sectionText, {color: colors.text}]}>
+              Wishlist
+            </Text>
+          </View>
+          <Icon
+            name="ios-chevron-forward"
+            size={25}
+            style={[styles.sectionIcon, {color: colors.text}]}
+          />
+        </TouchableOpacity>
 
-              <TouchableOpacity
-                style={styles.sectionBody}
-                onPress={() => {
-                  navigation.navigate('root');
-                }}>
-                <View style={styles.body}>
-                  <Icon
-                    name="ios-receipt-sharp"
-                    size={29}
-                    color={Colors.mediumGrey}
-                  />
-                  <Text style={styles.sectionText}>Order History</Text>
-                </View>
-                <Icon
-                  name="ios-chevron-forward"
-                  size={25}
-                  style={styles.sectionIcon}
-                />
-              </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.sectionBody}
+          onPress={() => {
+            navigation.navigate('root');
+          }}>
+          <View style={styles.body}>
+            <Icon
+              name="ios-receipt-sharp"
+              size={29}
+              color={Colors.mediumGrey}
+            />
+            <Text style={[styles.sectionText, {color: colors.text}]}>
+              Order History
+            </Text>
+          </View>
+          <Icon
+            name="ios-chevron-forward"
+            size={25}
+            style={[styles.sectionIcon, {color: colors.text}]}
+          />
+        </TouchableOpacity>
 
-              <TouchableOpacity
-                style={styles.sectionBody}
-                onPress={() => {
-                  navigation.navigate('root');
-                }}>
-                <View style={styles.body}>
-                  <Icon name="notifications" size={29} color={Colors.mediumGrey} />
-                  <Text style={styles.sectionText}>Notification</Text>
-                </View>
-                <Icon
-                  name="ios-chevron-forward"
-                  size={25}
-                  style={styles.sectionIcon}
-                />
-              </TouchableOpacity>
+        <TouchableOpacity
```

Comment:
If most of them are identical, then why not create a custom component and pass different props to them as needed. ?

Reply:
created a customProfileOption component.